pkgs/tools/networking/networkmanager_dmenu/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, glib, fetchFromGitHub, networkmanager, python3Packages
+, gobjectIntrospection, dmenu }:
+
+let inherit (python3Packages) python pygobject3;
+in stdenv.mkDerivation rec {
+  name = "networkmanager_dmenu-${version}";
+  version = "unstable-2017-04-13";
```

Comment:
Technically the other suggestion did not say whether `unstable` is a part of the version, it was just that it is desired in the resulting derivation name.